product-matrix.json

```diff
@@ -5,18 +5,20 @@
       "version": "3.1",
       "ideaProduct": "android-studio",
       "ideaVersion": "173.4670197",
-      "dartPluginVersion": "173.4700",
+      "dartPluginVersion": "181.4203.498",
```

Comment:
3.1 is still based on 2017.3, no? The Dart plugin version here should likely be `173.4700`.